htdocs/fourn/commande/card.php

```diff
@@ -1538,7 +1538,7 @@
 	print '<td>';
 	$usehourmin=0;
 	if (! empty($conf->global->SUPPLIER_ORDER_USE_HOUR_FOR_DELIVERY_DATE)) $usehourmin=1;
-	print $form->selectDate($datelivraison?$datelivraison:-1, 'liv_', $usehourmin, $usehourmin, '', "set");
+	print $form->selectDate($date_livraison?$date_livraison:-1, 'liv_', $usehourmin, $usehourmin, '', "set");
```

Comment:
Variable $date_livraison is not defined

Reply:
within pdf muscadet if you make a var_dump($object); you get back property "date_livraison" and not datelivraison. And it works with $date_livraison...

Reply:
I have fixed it by a forward.

Reply:
But here you are not into the code of pdf muscadet but into the page htdocs/fourn/commande/card.php
And in this page, variable $date_livraison is never defined generating a warning.
Please describe how to reproduce the bug. i can't reproduce on my environment.